src/client/utils/html.js

```diff
@@ -10,17 +10,21 @@ import { hasIsNotClosedFlag } from '../sandbox/node/document/writer';
 import * as urlResolver from './url-resolver';
 import INTERNAL_PROPS from '../../processing/dom/internal-properties';
 
-const FAKE_TAG_NAME_PREFIX  = 'fake_tag_name_';
-const FAKE_DOCTYPE_TAG_NAME = 'hammerhead_fake_doctype';
-const FAKE_TAG_NAME_RE      = new RegExp('(<\\/?)' + FAKE_TAG_NAME_PREFIX, 'ig');
-const WRAP_TAGS_RE          = /(<\/?)(html|head|body|table|tbody|tfoot|thead|tr|td|th|caption|colgroup)((?:\s[^>]*)?>)/ig;
-const WRAP_TAGS_TEMPLATE    = `$1${ FAKE_TAG_NAME_PREFIX }$2$3`;
-const WRAP_COL_TAG_RE       = /<(\/?(col)(?:[^A-Za-z_>][^>]*)?)>/ig;
-const WRAP_COL_TAG_TEMPLATE = `<${ FAKE_TAG_NAME_PREFIX }$2>$1</${ FAKE_TAG_NAME_PREFIX }$2>`;
+const FAKE_TAG_NAME_PREFIX    = 'hh_fake_tag_name_';
```

Comment:
`hh_fake_tag_name_` is prefix. For example:
`<table> -> <hh_fake_tag_name_table>`